cypress/e2e/home.cy.ts

```diff
@@ -10,34 +13,76 @@ describe('Visits the home page', () => {
         // TODO: clicking it should take us to '/'
     })
 
+    it('renders the map', () => {
+        // TODO
+        cy.get('[data-testid="map-of-japan"]')
+            .should('have.attr', 'src')
+            .and('not.be.empty')
+    })
+
     it('allows setting search fields', () => {
         cy.get('[data-testid="search-button"]').should('be.visible')
         // TODO
     })
 
-    it.skip('selects a language', () => {
+    it('can select "select a language"', () => {
+        cy.get('[data-testid="search-bar-language"]').trigger('click')
```

Comment:
Let's add some assertions to all the tests (or skip them if we don't have enough of the homepage working to finish)

Reply:
I understand. I'll skip this one. I guess in the following tests if I can select the languages, then this test by default should pass. I'll keep that in mind as I go forward